### PR TITLE
Feature#16/create footer header or sidebar

### LINKED ISSUE #16: #16 Create footer, header, or sidebar
# REQUIREMENTS
This task is for create a footer, header or sidebar for the app

# DESIGN
.-

# Acceptance Criteria
List all required acceptance criteria
- [ ] The menus should be displayed according to the user's role
- [ ] The header, footer or sidebar should be responsives

### PR DESCRIPTION
**Pull request by MiguelOcegueraM**
## Description: Why did you change/implement that?
We created all the footer and header design then we added SCSS

## Testing: Help me how can I test or look at the changes?

Use rails s

image

image
header

### CODE REVIEW

**Review by LPMendoza — APPROVED** [review]
It is a good advance, now let's go to do the functionality. Good work.

**Comment by LDVera** [comment]
All looks so good, continue working in the backend

File: reloj_checador_team03/app/views/test/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<form>
+<h3>Formulario ficticio</h3>
+  <div class="form-group">
+    <label for="exampleFormControlInput1">Email address</label>
+    <input type="email" class="form-control" id="exampleFormControlInput1" placeholder="name@example.com">
+  </div>
+  <div class="form-group">
+    <label for="exampleFormControlSelect1">Example select</label>
+    <select class="form-control" id="exampleFormControlSelect1">
+      <option>1</option>
```
Comment (LDVera) [inline reloj_checador_team03/app/views/test/index.html.erb:10]: maybe you can use and .each method with ruby here to create the options, i don't know if it works but can be a interesting experiment

**Review by LDVera — APPROVED** [review]
continue with the backend!

**Review by fego-dev — APPROVED** [review]
Good advance.
Let-s Go!

**Review by EdsonMorfin — APPROVED** [review]
looks good, Its a good advance

**Pull request merged**